src/IO/ReadBufferFromEncryptedFile.h

```diff
@@ -24,7 +25,7 @@ class ReadBufferFromEncryptedFile : public ReadBufferFromFileBase
     off_t seek(off_t off, int whence) override;
     off_t getPosition() override;
 
-    std::string getFileName() const override { return in->getFileName(); }
```

Comment:
`in->getFileName()` happened to be unreliable because it can somehow return an empty string, making logs difficult to read, for example:

```
2025.02.14 07:35:47.029576 [ 799199 ] {c9ccf3a8-08fc-444a-bc3a-4a09bc2bec8a} <Test> ReadBufferFromEncryptedFile: Decrypting bytes 0..44 from
2025.02.14 07:36:52.881181 [ 400737 ] {c9ccf3a8-08fc-444a-bc3a-4a09bc2bec8a} <Error> executeQuery: Code: 432. DB::Exception: Unknown codec family code: 245: While reading or decompressing  (position: 25, typename: DB::ReadBufferFromEncryptedFile) 
```